Source/WebCore/rendering/PathOperation.cpp

```diff
@@ -71,6 +80,13 @@ Ref<RayPathOperation> RayPathOperation::create(float angle, Size size, bool isCo
     return adoptRef(*new RayPathOperation(angle, size, isContaining, WTFMove(containingBlockBoundingRect), WTFMove(position)));
 }
 
+Ref<PathOperation> RayPathOperation::clone() const
+{
+    auto containingBlockBoundingRect = m_containingBlockBoundingRect;
+    auto position = m_position;
+    return adoptRef(*new RayPathOperation(m_angle, m_size, m_isContaining, WTFMove(containingBlockBoundingRect), WTFMove(position)));
```

Comment:
Do you gain anything with the local variables and the move here? Is it just because the method takes `&&`?

Reply:
Yes, I need copies to be made before they are moved.